### PR TITLE
BXMSDOC-4587-master: Update managing and monitoring business processes doc per changes in 7.5 

### PR DESCRIPTION
**Pull request by kaldesai**
- [Epic](https://issues.jboss.org/browse/BXMSDOC-4412)
- [JIRA](https://issues.jboss.org/browse/BXMSDOC-4588)
- [Managing & monitoring business processes in BC 7.5 RHPAM doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4587-ManagingandMonitoringBP-RHPAM/#managing-business-processes-bulk-actions-on-tasks-con)

Hey @nmirasch and @jstastny-cz, 
Can you please review and verify the new section **4.7. Bulk actions on tasks** that I have added in the **managing & monitoring business processes in BC** document? I have provided the RHPAM doc preview link and the respected JIRA.

P.S.-
I have already incorporated your suggested changes from the PR: https://github.com/kaldesai/kie-docs/pull/44
The content is same as of interacting with Processes & tasks document because both the docs have same BAPLs impact. 
The change is based on the following BAPLs:
- https://issues.jboss.org/browse/BAPL-1224
- https://issues.jboss.org/browse/BAPL-1362

Please let me know what do you think. 
Thank you.

### CODE REVIEW

File: assemblies/assembly_managing-and-monitoring-business-processes/main.adoc [hunk 1]
```diff
@@ -50,6 +49,12 @@ include::{enterprise-dir}/admin-and-config/tasks-default-filters-proc.adoc[level
 include::{enterprise-dir}/admin-and-config/viewing-task-variables-basic-filters-in-bc-proc.adoc[leveloffset=+2]
 include::{enterprise-dir}/admin-and-config/viewing-task-variables-advanced-filters-in-bc-proc.adoc[leveloffset=+2]
 include::{enterprise-dir}/admin-and-config/manage-service-tasks-proc.adoc[leveloffset=+2]
+include::{enterprise-dir}/admin-and-config/managing-business-processes-bulk-actions-on-tasks-con.adoc[leveloffset=+2]
```
Comment (jstastny-cz) [inline assemblies/assembly_managing-and-monitoring-business-processes/main.adoc]: Move the section one only after "4.8. User task administration", i.e. under line
```
include::{jbpm-dir}/BPMN2/user-task-admin-con.adoc[leveloffset=+2]
```

Currently we're talking about bulk actions even before we introduce the basic operations for tasks.

Comment (kaldesai) [reply assemblies/assembly_managing-and-monitoring-business-processes/main.adoc]: @jstastny-cz, Ack. Thanks!

**Comment by kaldesai** [comment]
Hey @jstastny-cz, I have incorporated your suggestion. Can you please review and verify the doc once again? I have updated the rendered output link.

**Comment by kaldesai** [comment]
Hey @emmurphy1, I have incorporated your suggestion. Can you please review and verify the doc once again? I have updated the rendered output link.

**Pull request merged**